src/components/IntroductionVideo.svelte

```diff
@@ -0,0 +1,42 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+ 
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import StandardDialog from './dialogs/StandardDialog.svelte';
+  import { t } from '../i18n';
+  import { state } from '../script/stores/uiStore';
+  import StandardButton from './StandardButton.svelte'
+
+  let isOpen = true;
+  $: modalVisibility = isOpen;
+  console.log("MODAL VISIBILITY: ", modalVisibility);
+</script>
+
+<StandardDialog {isOpen} onClose={() => (isOpen = false)}>
+  <div class="w-150">
+    <p class="text-2xl mb-2 font-semibold">
+      {$t('content.introductionVideo.heading')}
+    </p>
+    <div class="flex items-center justify-center">
+      <!-- We have selected the video's loading progress as the loading criterion -->
+      <video
+        on:canplaythrough={() => ($state.isLoading = false)}
+        class="w-full"
+        controls
+        width="550"
+        poster="imgs/data-trainer-thumpnail.png">
+        <source
+          src="https://datatraener0dr0media-euno.streaming.media.azure.net/18233c69-2bc3-4b1b-9e2d-249e37b56307/Ultrabit_01_Introvideo_Datatræneren.mp4"
+          type="video/mp4" />
+      </video>
+    </div>
+    <p class="pt-5">{$t('content.introductionVideo.description')}</p>
+    <div class="ml-110">
```

Comment:
It would be good to remove this `div`. Currently it is making the button text wrap onto the next line. It would be good if we could modify the `StandardButton` to set its alignment in its containing `div` using a prop. I've had a quick experiment with using `place-items-start`, `place-items-center` and `place-items-end`, but haven't quite got it to work.